styles/style.css

```
@@ -25,35 +32,78 @@ body {
   background-color: #ffffff;
   color: #000000;
 }
-.button{
-  color: #FFFFFF;
-  background-color:#83B3D3;
+.main {
+  flex-grow: 1;
+
+}
+.button {
+  color: #ffffff;
   font-weight: bold;
   text-transform: uppercase;
   text-decoration: none;
   font-family: inherit;
-  font-size: 20px;
-  line-height: 26px;
+}
+.visually-hidden {
+  visibility: hidden;
+  display: none;
+  margin: 0px;
+  padding: 0px;
+}
+.box-сonteiner{
+  display: flex;
+  width: 1200px;
+  margin: 0 auto;
+  align-content: center;
 }
 /*header*/
+
+.navigation {
+  margin: 0px;
+  align-content: center;
+  display: flex;
+  width: 1057px;
+  margin: 0 auto;
+}
+.navigation-list {
+  list-style-type: none;
+  display: flex;
+  width: 412px;
+  margin-right: auto;
+}
+.logo-pic {
+  width: 138px;
+  align-content: center;
+}
+.navigation-user {
+  list-style-type: none;
+  display: flex;
+  width: 300px;
+  margin-left: 120px;
+}
```

Поскольку на это элементе сразу два класса
```suggestion
.navigation-user {
  	justify-content: space-between;
	width: 250px;
	gap: 0;
}
```